kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/SchemaManager.java

```diff
@@ -495,6 +495,9 @@ TableInfo constructTableInfo(TableId table, com.google.cloud.bigquery.Schema big
   }
 
   private com.google.cloud.bigquery.Schema getBigQuerySchema(Schema kafkaKeySchema, Schema kafkaValueSchema) {
+    if (kafkaValueSchema == null) {
+      return com.google.cloud.bigquery.Schema.of();
```

Comment:
Does this just return an empty schema? Are there any consequences with upsert w.r.t to any kind of schema evolution. In other words, I'm not sure if an empty schema changes any behavior vs a null value.

Reply:
With null value, task would fail.
With empty value, yeah, there is consequence, as we only look at the schema of the last record inside the batch. With this change, we need to enable unionization to make it work.

Reply:
We're trying to accommodate a pretty specific case here (schema unionization is enabled, and the last record in the first batch it sees for a table is a tombstone record), but this change is fairly general and will have an effect whenever the connector sees a schemaless record when any kind of DDL is required. Should we be more specific in when we permit this, and fail fast otherwise?

For example, if schema unionization is disabled and the record doesn't have a schema associated with it, is there any benefit to returning an empty schema here?

Reply:
Yes, in that case we should throw. Updated.